server/server.go

```diff
@@ -1467,6 +1470,48 @@ func (s *Server) fetchAccount(name string) (*Account, error) {
 	return acc, nil
 }
 
+func (s *Server) setupOCSPStapleStoreDir() error {
+	opts := s.getOpts()
+	storeDir := opts.StoreDir
+	if storeDir == _EMPTY_ {
+		s.Warnf("OCSP Stapling disk cache is disabled (missing 'store_dir')")
+		return nil
+	}
+	storeDir = filepath.Join(storeDir, defaultOCSPStoreDir)
+	if stat, err := os.Stat(storeDir); os.IsNotExist(err) {
+		if err := os.MkdirAll(storeDir, os.FileMode(0750)); err != nil {
```

Comment:
I think there is a constant now in the server (for JetStream) to use instead: `defaultDirPerms  = os.FileMode(0750)`. Not sure if you have rebased recently.

Reply:
ah yes now that I rebased can use this